install.sh

```diff
@@ -270,7 +270,7 @@ check_path() {
 	then
 		echo  #spacer
 
-		if [ -w /usr/local/bin ];
+		if [ -w /usr/local/bin ] || [ -w /usr/local ] || [ -w /usr ] || [ -w / ];
```

Comment:
I'm not sure this is precisely correct, but you'd have to do something very esoteric to break it.
```sh
sudo mkdir -p /usr/local/bin
chmod 555 /usr/local/bin
chmod 777 /usr
```

Reply:
Works!

Reply:
Works!